writers/cassandra/README.md

```diff
@@ -0,0 +1,72 @@
+# Cassandra writer
+
+Cassandra writer provides message repository implementation for Cassandra.
+
+## Configuration
+
+The service is configured using the environment variables presented in the
+following table. Note that any unset variables will be replaced with their
+default values.
+
+| Variable                        | Description                                 | Default               |
+|---------------------------------|---------------------------------------------|-----------------------|
+| MF_NATS_URL                     | NATS instance URL                           | nats://localhost:4222 |
+| MF_CASSANDRA_WRITER_PORT        | Service HTTP port                           | 8180                  |
+| MF_CASSANDRA_WRITER_DB_CLUSTER  | Cassandra cluster comma separated addresses | 127.0.0.1             |
```

Comment:
Rename default to `localhost` if possible.

Reply:
I would like to keep this IP address because this is array of cluster addresses. These addresses will usually be added as IP addresses.